static/app/utils/discover/fields.tsx

```diff
@@ -15,6 +15,7 @@ export type Sort = {
 // Can be parsed into a Column using explodeField()
 export type Field = {
   field: string;
+  alias?: string;
```

Comment:
What does alias mean here or how should it be used? Also, can we just make it required :)?

Reply:
We can't make it required because not all visualizations support alias yet... we will improve it in the future 😉

Reply:
Ok, but I wouldnt mind a comment about what are aliases used for?

Reply:
@JonasBa done 😉